imap_processing/quality_flags.py

```diff
@@ -0,0 +1,46 @@
+"""Bitwise flagging."""
+
+from enum import IntFlag
+
+
+class CommonFlags(IntFlag):
+    """Common quality flags."""
+
+    NONE = 0x0
+    INF = 2**0  # bit 0, Infinite value
+    NEG = 2**1  # bit 1, Negative value
+
+
+class ENAFlags(IntFlag):
+    """Common ENA flags."""
+
+    BADSPIN = 2**2  # bit 2, Bad spin
```

Comment:
Just to be sure, this `2**2` is 4. If we continue using this kind of power increase, will we run out of values soon or is it ok?
```suggestion
    BADSPIN = 2**2  # 4, Bad spin
```